tests/common/src/lib.rs

```diff
@@ -260,7 +261,7 @@ struct Journal<'a> {
 impl<'a> JournalHandler for Journal<'a> {
     fn message_dispatched(&mut self, outcome: DispatchOutcome) {
         match outcome {
-            DispatchOutcome::Success(_) => {}
+            DispatchOutcome::Success(_) | DispatchOutcome::Skip(_) => {}
```

Comment:
Why to not combine all empty branches into one?
```suggestion
            _ => (),
```